packages/graphic-walker/src/computation/index.ts

```diff
@@ -163,6 +163,74 @@ export const fieldStat = async (service: IComputationFunction, field: IField, op
     };
 };
 
+export async function getDistinctValues(service: IComputationFunction, field: string) {
+    const COUNT_ID = `count_${field}`;
```

Comment:
do not use count, use unique. Count has other semantic meanings in graphic-walker.

Besides, I am not quite sure if the prefix will only be used in this function.